packages/infra/lib/api-stack.ts

```diff
@@ -1080,6 +1095,64 @@ export class APIStack extends Stack {
     return documentDownloaderLambda;
   }
 
+  private setupBulkUrlSigningLambda(ownProps: {
```

Comment:
nit: could try to name this in a way that resembles the lambda file name, maybe `setupDocumentBulkSigner`?